store.go

```diff
@@ -628,12 +624,6 @@ func (s *store) PutLayer(id, parent string, names []string, mountLabel string, w
 	if modified, err := rlstore.Modified(); modified || err != nil {
 		rlstore.Load()
 	}
-	ristore.Lock()
-	defer ristore.Unlock()
-	defer ristore.Touch()
```

Comment:
We're okay not calling Touch() if the set of data that the store object keeps isn't being changed, which appears to be true here.